src/doom/p_maputl.c

```diff
@@ -529,6 +529,121 @@ P_BlockThingsIterator
 	if (!func( mobj ) )
 	    return false;
     }
+
+    // [crispy] Blockmap bug fix - add other mobjs from
+    // surrounding blocks that overlap this one
+    if (crispy->blockmapfix)
```

Comment:
That's be `critical->blockmapfix` here to make sure it's not applied during demos and network game.